charts/atlantis/values.yaml

```diff
@@ -260,7 +260,7 @@ statefulSet:
   labels: {}
   securityContext:
     fsGroup: 1000
```

Comment:
We need to discuss if this is the right thing to do.
We made changes to Alpine and Debian to match the UID recently and we need to keep backwards compatibility
this could affect users that are not enforcing the default.

@GenPage @GMartinez-Sisti @nitrocode

Reply:
This is going to be closed. There’s another PR fixes the uid to be 100 on both -> https://github.com/runatlantis/atlantis/pull/4304. Please let us know what you think.

Reply:
Prior to merging #4304, is this fix #359 even needed when the values can be overridden for `runAsUser`?

Reply:
If runatlantis/atlantis#4304 gets merged in its current state this won't be necessary anymore no.